index.js

```diff
@@ -157,10 +157,59 @@ function displayFooter() {
 }
 
 function activateSortBtns() {
-  sortBtns.forEach((btn) => btn.addEventListener("click", handleSortBtnClick));
+  sortBtns.forEach((btn) => {
+    btn.addEventListener("click", handleSortBtnClick);
+    btn.sortingState = null;
```

Comment:
I'd change the naming like this:

```suggestion
    btn.sortingOrder = null;
```

Reply:
Makes sense, will be done